src/Stratis.Bitcoin/Configuration/NodeSettings.cs

```diff
@@ -215,7 +215,7 @@ public static NodeSettings Default(Network network = null, ProtocolVersion proto
 
             nodeSettings.RequireStandard = config.GetOrDefault("acceptnonstdtxn", !(nodeSettings.RegTest || nodeSettings.Testnet));
             nodeSettings.MaxTipAge = config.GetOrDefault("maxtipage", DefaultMaxTipAge);
-            nodeSettings.ApiUri = config.GetOrDefault("apiuri", new Uri("http://localhost:37220"));
+            nodeSettings.ApiUri = config.GetOrDefault("apiuri", (nodeSettings.Network == Network.StratisMain || nodeSettings.Network == Network.StratisTest || nodeSettings.Network == Network.StratisRegTest) ? new Uri("http://localhost:37221") : new Uri("http://localhost:37220"));
```

Comment:
indeed, good spot.
Can you please move the endpoints to be constants? look in StratisBitcoinFullNode\src\Stratis.BreezeD\Program.cs

Reply:
While we have a more wider plan to refactor Network for now I would actually say we add a new property to Network and call it ApiEndpoint and set it at network level.

Reply:
@andrasfuchs I suggest you leave it for now, I'm working on a task that's moving all of these constants and default values around.

Reply:
Alright, cool! I'll keep my eyes open for those changes then :)